Code/Source/VehicleDynamics/DriveModels/SimplifiedDriveModel.cpp

```diff
@@ -24,7 +24,11 @@ namespace VehicleDynamics
             serialize->Class<SimplifiedDriveModel, DriveModel>()
                 ->Version(1)
                 ->Field("SteeringPID", &SimplifiedDriveModel::m_steeringPid)
-                ->Field("SpeedPID", &SimplifiedDriveModel::m_speedPid);
+                ->Field("SpeedPID", &SimplifiedDriveModel::m_speedPid)
+                ->Field("MaxSpeedImpulse", &SimplifiedDriveModel::maxSpeedImpulse)
+                ->Field("ShowSteeringDebugInfo", &SimplifiedDriveModel::showSteeringDebugInfo)
```

Comment:
Could you please move these debug fields (and debugs themselves) to a separate PR? This is different than limiting impulse.

Reply:
Perhaps one is enough? Just one variable "show debug values". Perhaps actually showing them in a component fields (e.g. so that the user does not need to dig in the logs) would be good? 

We need to have 3 fields for speed and 3 for steering, a nice layout of 3x2 would be good to present values.

That is, if you believe it would be useful for users in general. If it is only for us, then move it out to a draft PR which we will not be merging.